turbopack/crates/turbopack-ecmascript/src/references/esm/export.rs

```diff
@@ -497,12 +498,17 @@ impl CodeGenerateable for EsmExports {
                     "(() => { throw new Error(\"Failed binding. See build errors!\"); })" as Expr,
                 )),
                 EsmExport::LocalBinding(name, mutable) => {
+                    let local = if name == "default" {
+                        magic_identifier::mangle("default export").into()
+                    } else {
+                        name.clone()
+                    };
```

Comment:
This could be a bit more efficient by avoiding the Arc

```suggestion
                    let local = if name == "default" {
                        magic_identifier::mangle("default export").into()
                    } else {
                        Cow::Borrowed(name.as_str())
                    };
```